packages/itwinui-react/src/core/ColorPicker/ColorInputPanel.tsx

```diff
@@ -450,33 +477,57 @@ export const ColorInputPanel = React.forwardRef((props, ref) => {
     </>
   );
 
-  const labelId = useId();
+  const fallbackLabelId = useId();
+  const labelId = panelLabelProps?.id ?? fallbackLabelId;
 
   return (
     <Box
+      as='div'
       className={cx('iui-color-input-wrapper', className)}
       ref={ref}
       {...rest}
     >
-      <Box className='iui-color-picker-section-label' id={labelId}>
+      <Box
+        as='div'
+        {...panelLabelProps}
+        className={cx(
+          'iui-color-picker-section-label',
+          panelLabelProps?.className,
+        )}
+        id={labelId}
+      >
         {showAlpha && currentFormat !== 'hex'
           ? currentFormat.toUpperCase() + 'A'
           : currentFormat.toUpperCase()}
       </Box>
-      <Box className='iui-color-input'>
+      <Box
+        as='div'
+        {...inputContainerProps}
+        className={cx('iui-color-input', inputContainerProps?.className)}
+      >
         {allowedColorFormats.length > 1 && (
           <IconButton
-            styleType='borderless'
-            onClick={swapColorFormat}
-            size='small'
-            label='Switch format'
+            size={'small'}
+            styleType={'borderless'}
+            label={'Switch format'}
```

Comment:
nit: none of these props should have curly braces `{}` added. Can you try to review your own PR to make sure there are no such stray/leftover changes?

Reply:
I have removed the braces.